apps/teams-perf-test-app/webpack.config.js

```diff
@@ -43,5 +43,6 @@ module.exports = {
     https: true,
     disableHostCheck: true,
   },
+  performance: { hints: false },
```

Comment:
Is this a new behavior we want?

Reply:
Yes. We had a thread going in our Engineering channel about how we expect the Teams package to be larger than webpack is used to, so we concluded to turn the hints off about performance. If this makes you feel too icky, we can also totally explore setting a higher number that webpack should warn us at, although my issue was that I couldn't decide on just what number we should be setting it at.